app/build.gradle

```diff
@@ -56,6 +57,7 @@ dependencies {
     implementation libs.commons.validator
 
     // Import the Firebase BoM
-    implementation platform(libs.firebase.bom)
-    implementation libs.firebase.crashlytics
+    // Comment out Firebase dependencies temporarily
+    // implementation platform(libs.firebase.bom)
+    // implementation libs.firebase.crashlytics
```

Comment:
Revert these lines before merge (just for my own testing so I didn't have to set up Google Services / Firebase).